### DISCUSSION TITLE
Teoretical question OGXbox support?

### DISCUSSION THREAD

**Discussion by patrikmckane** [darthcloud/BlueRetro#928] [General]
Hi. First of all I apreciate your work on the blueretro. Teoreticky question can or Will the blueretro support the og xbox? I know there is the ogx360 but I really dont Like its setup. Thanks for answer

**Comment by konwektor** [comment]
Blueretro  can not be used alone for ogxbox because
when you connect esp32 to a pc through USB there is USB-UART bridge and esp32 will be shown as a serialPort on Device manager. Same for ogxbox. Esp32 can emulate HID device only over bluetooth.
Instead try Blueretro forks for ogxbox - using esp32 plus ogx360 by Ryzee119 arduino modules:
Orginal author :
https://github.com/netham45/BlueRetro
Forks of netham45 work:
https://github.com/jackrayner/BlueRetro
https://davidxg.blogspot.com/2023/11/blueogx.html?m=1
Mine:
https://github.com/konwektor/BlueRetro

Reddit links:
https://www.reddit.com/r/originalxbox/comments/z5kqiu/ogx360_blueretro/
https://www.reddit.com/r/originalxbox/comments/18s5bx4/wireless_playing_for_few_bucks/
https://www.reddit.com/r/originalxbox/comments/16jkqlp/bluetooth_on_the_xbox_ogx360_blueretro/

**Reply by patrikmckane** [reply]
I dont want to use ogxbox. First of all its big bunky and not very convenient. I want something Like vlueretro. Plug to controller port a dongle and be ready not Like ogx360.

**Reply by konwektor** [reply]
Like I said esp32 can not emulate HID device over usb - so blueretro without extra module can not be used on ogxbox. 
If you find solutionfor that, share with Rest ppl.

**Reply by patrikmckane** [reply]
As I said I dont wanna use ogxbox360. So I need to find another solution.

**Reply by konwektor** [reply]
Try this one https://github.com/wiredopposite/OGX-Mini
Raspberry pi pico with usb host- usb device parser. Just plug in konsole port and plug into pi pico  any modern wired usb gamepad

**Comment by darthcloud** [comment]
Its coming this year 😊

https://x.com/darthcloud64/status/1772816124507263248?s=20

**Reply by patrikmckane** [reply]
How much Will it Cost?
